<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BMGTexToolClass</class>
 <widget class="QMainWindow" name="BMGTexToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>BMGTexTool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="BMGTexTool.qrc">
    <normaloff>:/BMGTexTool/Resources/TX.png</normaloff>:/BMGTexTool/Resources/TX.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/*
	Copyright 2013 Emanuel Claesson

	Licensed under the Apache License, Version 2.0 (the &quot;License&quot;);
	you may not use this file except in compliance with the License.
	You may obtain a copy of the License at

		http://www.apache.org/licenses/LICENSE-2.0

	Unless required by applicable law or agreed to in writing, software
	distributed under the License is distributed on an &quot;AS IS&quot; BASIS,
	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
	See the License for the specific language governing permissions and
	limitations under the License.
*/

/*
	COLOR_DARK     = #1e1e1e
	COLOR_MEDIUM   = #2d2d30
	COLOR_MEDLIGHT = #333337
	COLOR_LIGHT    = #FFFFFF
	COLOR_ACCENT   = #3f3f46
*/

* {
	background: #1e1e1e;
	color: #FFFFFF;
	border: 1px solid #333337;
}

QWidget::item:selected {
	background: #3f3f46;
}

QCheckBox, QRadioButton {
	border: none;
}

QRadioButton::indicator, QCheckBox::indicator {
	width: 13px;
	height: 13px;
}

QRadioButton::indicator::unchecked, QCheckBox::indicator::unchecked {
	border: 1px solid #333337;
	background: none;
}

QRadioButton::indicator:unchecked:hover, QCheckBox::indicator:unchecked:hover {
	border: 1px solid #FFFFFF;
}

QRadioButton::indicator::checked, QCheckBox::indicator::checked {
	border: 1px solid #333337;
	background: #333337;
}

QRadioButton::indicator:checked:hover, QCheckBox::indicator:checked:hover {
	border: 1px solid #FFFFFF;
	background: #FFFFFF;
}

QGroupBox {
	margin-top: 6px;
}

QGroupBox::title {
	top: -7px;
	left: 7px;
}

QScrollBar {
	border: 1px solid #686868;
	background: #1e1e1e;
}

QScrollBar:horizontal {
	height: 15px;
	margin: 0px 0px 0px 32px;
}

QScrollBar:vertical {
	width: 15px;
	margin: 32px 0px 0px 0px;
}

QScrollBar::handle {
	background: #2d2d30;
	border: 1px solid #333337;
}

QScrollBar::handle:horizontal {
	border-width: 0px 1px 0px 1px;
}

QScrollBar::handle:vertical {
	border-width: 1px 0px 1px 0px;
}

QScrollBar::handle:horizontal {
	min-width: 20px;
}

QScrollBar::handle:vertical {
	min-height: 20px;
}

QScrollBar::add-line, QScrollBar::sub-line {
	background:#2d2d30;
	border: 1px solid #333337;
	subcontrol-origin: margin;
}

QScrollBar::add-line {
	position: absolute;
}

QScrollBar::add-line:horizontal {
	width: 15px;
	subcontrol-position: left;
	left: 15px;
}

QScrollBar::add-line:vertical {
	height: 15px;
	subcontrol-position: top;
	top: 15px;
}

QScrollBar::sub-line:horizontal {
	width: 15px;
	subcontrol-position: top left;
}

QScrollBar::sub-line:vertical {
	height: 15px;
	subcontrol-position: top;
}

QScrollBar:left-arrow, QScrollBar::right-arrow, QScrollBar::up-arrow, QScrollBar::down-arrow {
	border: 1px solid #333337;
	width: 3px;
	height: 3px;
}

QScrollBar::add-page, QScrollBar::sub-page {
	background: none;
}

QAbstractButton:hover {
	background: #2d2d30;
}

QAbstractButton:pressed {
	background: #333337;
}

QAbstractItemView {
	show-decoration-selected: 1;
	selection-background-color: #3f3f46;
	selection-color: #FFFFFF;
	alternate-background-color: #2d2d30;
}

QHeaderView {
	border: 1px solid #333337;
}

QHeaderView::section {
	background: #1e1e1e;
	border: 1px solid #333337;
	padding: 4px;
}

QHeaderView::section:selected, QHeaderView::section::checked {
	background: #2d2d30;
}

QTableView {
	gridline-color: #333337;
}

QTabBar {
	margin-left: 2px;
}

QTabBar::tab {
	border-radius: 0px;
	padding: 4px;
	margin: 4px;
}

QTabBar::tab:selected {
	background: #2d2d30;
}

QComboBox::down-arrow {
	border: 1px solid #333337;
	background: #2d2d30;
}

QComboBox::drop-down {
	border: 1px solid #333337;
	background: #2d2d30;
}

QComboBox::down-arrow {
	width: 3px;
	height: 3px;
	border: 1px solid #333337;
}

QAbstractSpinBox {
	padding-right: 15px;
}

QAbstractSpinBox::up-button, QAbstractSpinBox::down-button {
	border: 1px solid #333337;
	background: #2d2d30;
	subcontrol-origin: border;
}

QAbstractSpinBox::up-arrow, QAbstractSpinBox::down-arrow {
	width: 3px;
	height: 3px;
	border: 1px solid #333337;
}

QSlider {
	border: none;
}

QSlider::groove:horizontal {
	height: 5px;
	margin: 4px 0px 4px 0px;
}

QSlider::groove:vertical {
	width: 5px;
	margin: 0px 4px 0px 4px;
}

QSlider::handle {
	border: 1px solid #333337;
	background: #2d2d30;
}

QSlider::handle:horizontal {
	width: 15px;
	margin: -4px 0px -4px 0px;
}

QSlider::handle:vertical {
	height: 15px;
	margin: 0px -4px 0px -4px;
}

QSlider::add-page:vertical, QSlider::sub-page:horizontal {
	background: #3f3f46;
}

QSlider::sub-page:vertical, QSlider::add-page:horizontal {
	background: #2d2d30;
}

QLabel {
	border: none;
}

QProgressBar {
	text-align: center;
}

QProgressBar::chunk {
	width: 1px;
	background-color: #3f3f46;
}

QMenu::separator {
	background: #2d2d30;
}</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionBatch"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="separator"/>
    <addaction name="actionReset"/>
   </widget>
   <widget class="QMenu" name="menuMipmaps">
    <property name="title">
     <string>Mipmaps</string>
    </property>
    <addaction name="actionGenerateMip"/>
    <addaction name="actionClean"/>
    <addaction name="separator"/>
    <addaction name="actionUpperMip"/>
    <addaction name="actionLowerMip"/>
   </widget>
   <widget class="QMenu" name="menuCubeMap">
    <property name="title">
     <string>CubeMap</string>
    </property>
    <addaction name="actionGenerateCube"/>
    <addaction name="separator"/>
    <addaction name="actionPx"/>
    <addaction name="actionNx"/>
    <addaction name="actionPy"/>
    <addaction name="actionNy"/>
    <addaction name="actionPz"/>
    <addaction name="actionNz"/>
   </widget>
   <widget class="QMenu" name="menu3D">
    <property name="title">
     <string>3D</string>
    </property>
    <addaction name="actionGenerate3D"/>
    <addaction name="separator"/>
    <addaction name="actionUpper3D"/>
    <addaction name="actionLower3D"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuCompress">
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="title">
     <string>Compress</string>
    </property>
    <addaction name="actionASTC"/>
    <addaction name="actionS3TC"/>
    <addaction name="actionETC2"/>
    <addaction name="separator"/>
    <addaction name="actionCompress"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuCompress"/>
   <addaction name="menuMipmaps"/>
   <addaction name="menuCubeMap"/>
   <addaction name="menu3D"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>368</width>
     <height>292</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="windowTitle">
    <string>Texture Tag Sheet</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <column>
        <property name="text">
         <string>Tag</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Area</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="BMGTexTool.qrc">:/BMGTexTool/Resources/logo.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open Image Files</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="BMGTexTool.qrc">
     <normaloff>:/BMGTexTool/Resources/save.png</normaloff>:/BMGTexTool/Resources/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save Bmg File</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="BMGTexTool.qrc">
     <normaloff>:/BMGTexTool/Resources/close.png</normaloff>:/BMGTexTool/Resources/close.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit Application</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="BMGTexTool.qrc">
     <normaloff>:/BMGTexTool/Resources/reset.png</normaloff>:/BMGTexTool/Resources/reset.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Zoom Reset</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionUpperMip">
   <property name="text">
    <string>Upper</string>
   </property>
   <property name="toolTip">
    <string>Upper Mipmap</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="actionLowerMip">
   <property name="text">
    <string>Lower</string>
   </property>
   <property name="toolTip">
    <string>Lower Mipmap</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
  <action name="actionGenerateMip">
   <property name="icon">
    <iconset resource="BMGTexTool.qrc">
     <normaloff>:/BMGTexTool/Resources/generate.png</normaloff>:/BMGTexTool/Resources/generate.png</iconset>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
   <property name="toolTip">
    <string>Generate Mipmap</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="actionClean">
   <property name="text">
    <string>Clean</string>
   </property>
   <property name="toolTip">
    <string>Clean Mipmap</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="actionGenerateCube">
   <property name="icon">
    <iconset resource="BMGTexTool.qrc">
     <normaloff>:/BMGTexTool/Resources/generate.png</normaloff>:/BMGTexTool/Resources/generate.png</iconset>
   </property>
   <property name="text">
    <string>Create Template</string>
   </property>
   <property name="toolTip">
    <string>Generate CubeMap</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="actionPx">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Px</string>
   </property>
   <property name="toolTip">
    <string>Px Side</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="actionNx">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Nx</string>
   </property>
   <property name="toolTip">
    <string>Nx Side</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="actionPy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Py</string>
   </property>
   <property name="toolTip">
    <string>Py Side</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="actionNy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ny</string>
   </property>
   <property name="toolTip">
    <string>Ny Side</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="actionPz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pz</string>
   </property>
   <property name="toolTip">
    <string>Pz Side</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="actionNz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Nz</string>
   </property>
   <property name="toolTip">
    <string>Nz Side</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="actionUpper3D">
   <property name="text">
    <string>Upper</string>
   </property>
   <property name="toolTip">
    <string>Upper Texture Layer</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Up</string>
   </property>
  </action>
  <action name="actionGenerate3D">
   <property name="icon">
    <iconset resource="BMGTexTool.qrc">
     <normaloff>:/BMGTexTool/Resources/generate.png</normaloff>:/BMGTexTool/Resources/generate.png</iconset>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
   <property name="toolTip">
    <string>Generate 3D Texture Volume</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="actionLower3D">
   <property name="text">
    <string>Lower</string>
   </property>
   <property name="toolTip">
    <string>Lower Texture Layer</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Down</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="BMGTexTool.qrc">
     <normaloff>:/BMGTexTool/Resources/TX.png</normaloff>:/BMGTexTool/Resources/TX.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close Image</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="actionBatch">
   <property name="text">
    <string>Batch</string>
   </property>
  </action>
  <action name="actionASTC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ASTC</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Shift+A</string>
   </property>
  </action>
  <action name="actionS3TC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S3TC</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Shift+S</string>
   </property>
  </action>
  <action name="actionETC2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ETC2</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Shift+E</string>
   </property>
  </action>
  <action name="actionCompress">
   <property name="icon">
    <iconset resource="BMGTexTool.qrc">
     <normaloff>:/BMGTexTool/Resources/generate.png</normaloff>:/BMGTexTool/Resources/generate.png</iconset>
   </property>
   <property name="text">
    <string>Compress</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BMGTexTool.qrc"/>
 </resources>
 <connections/>
</ui>
